NOTICES

```diff
@@ -5,7 +5,7 @@ This file details additional third party software license agreements
 and third party notices and information that are required 
 to be reproduced for the following programs:
    
-Operator for IBM Block Storage CSI Driver version 1.8.0
+Operator for IBM Block Storage CSI Driver version 1.9.0

```

Comment:
could you open a ticket to sync the operator name for future eLicense? it [was changed in 1.7.0](https://github.com/IBM/ibm-block-csi-driver/pull/320#pullrequestreview-908188184)...

Reply:
CSI-4010